File: python/06_arrays.py
n = int(input("Enter number of elements: "))

arr = []
print(f"Enter {n} elements:")
for i in range(n):
    arr.append(int(input()))

print(f"\nArray elements: {arr}")
print(f"Sum = {sum(arr)}")
print(f"Average = {sum(arr) / n}")

File: cpp/06_arrays.cpp
#include <iostream>
using namespace std;

int main() {
    int n, sum = 0;
    
    cout << "Enter number of elements: ";
    cin >> n;
    
    int arr[n];
    
    cout << "Enter " << n << " elements:" << endl;
    for (int i = 0; i < n; i++) {
        cin >> arr[i];
        sum += arr[i];
    }
    
    cout << "\nArray elements: ";
    for (int i = 0; i < n; i++) {
        cout << arr[i] << " ";
    }
    
    cout << "\nSum = " << sum << endl;
    cout << "Average = " << (float)sum / n << endl;
    
    return 0;
}
